kasten/k10-minimum-retention/patched.yaml

```diff
@@ -0,0 +1,40 @@
+kind: Policy
+apiVersion: config.kio.kasten.io/v1alpha1
+metadata:
+  name: hourly-policy
+  namespace: kasten-io
+    appPriority: Mission-Critical
```

Comment:
This line is causing tests to fail because it's invalid. It looks like a label and so should either be placed under `metadata.labels` or simply removed.